src/lang/ts/intermock.ts

```diff
@@ -321,6 +346,78 @@ function processArrayPropertyType(
   }
 }
 
+/**
+ * Process an array definition.
+ *
+ * @param node Node being processed
+ * @param output The object outputted by Intermock after all types are mocked
+ * @param property Output property to write to
+ * @param typeName Type name of property
+ * @param kind TS data type of property type
+ * @param sourceFile TypeScript AST object compiled from file data
+ * @param options Intermock general options object
+ * @param types Top-level types of interfaces/aliases etc.
+ */
+function processUnionPropertyType(
+    node: ts.PropertySignature, output: Output, property: string,
+    typeName: string, kind: ts.SyntaxKind, sourceFile: ts.SourceFile,
+    options: Options, types: Types) {
+  // @ts-ignore
```

Comment:
Can you remove these `ts-ignore` comments?

Reply:
Done